qa-tests-backend/src/test/groovy/ProcessVisualizationTest.groovy

```diff
@@ -88,7 +88,8 @@ class ProcessVisualizationTest extends BaseSpecification {
     @Tag("BAT")
     @Tag("RUNTIME")
     // TODO(ROX-16461): Fails under AKS
-    @IgnoreIf({ Env.CI_JOB_NAME.contains("aks-qa-e2e") })
+    // TODO(ROX-23915): Fails under GKE
+    @IgnoreIf({ Env.CI_JOB_NAME.contains("aks-qa-e2e") || Env.CI_JOB_NAME.contains("gke") })
```

Comment:
It deserves a check that `gke` matches only `gke*qa-e2e` tests and does not match anything else. Although one can assume that with 99% of confidence, I don't think burden of proving this is on reviewers.

Pivoting this question a little bit, the unconfidence is that neither I nor you seem to have looked into what _actually_ comes in `$CI_JOB_NAME` environment variable and `gke-qa-e2e` was a _safer_ guess because it followed the pattern of `aks-qa-e2e` introduced by someone else.